website/views.py

```diff
@@ -3503,116 +3503,24 @@ def safe_redirect(request: HttpRequest):
     fallback_url = f"{request.scheme}://{request.get_host()}/"
     return redirect(fallback_url)
 
-# class CreateIssue(CronJobBase):
-#     RUN_EVERY_MINS = 1
-
```

Comment:
This code is a future feature please revert

Reply:
Ohh sorry didn't knew about it. Revert done 0ec0ea76c7abbcd29983cede000687c627622b26